ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -40,6 +40,11 @@ images:
       as:
       - registry.ci.openshift.org/openshift/release:golang-1.15
   to: assisted-service
+- dockerfile_literal: |
+    FROM centos:8
+    COPY . .
```

Comment:
Isn't `deploy` directory enough?

Reply:
no
you don't remember we need ``setup_env.sh`` as well?